src/store/traction/ont/pools/actions.js

```diff
@@ -367,15 +368,16 @@ export default {
    */
   // For the component, the included relationships are not required
   // However, the functionality does not appear to work without them
+  // no unit tests?
```

Comment:
Worth adding unit tests here?

Reply:
It is covered bye e2e tests. We can add some at some point but I needed to stop somewhere.